.github/workflows/push-mimir-build-image.yml

```diff
@@ -17,6 +17,9 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Dependabot doesn't have `secrets.GITHUB_TOKEN`, why will this work?

Reply:
see reply above ☝️